sklearn/compose/_column_transformer.py

```diff
@@ -936,6 +942,19 @@ def _get_transformer_list(estimators):
     return transformer_list
 
 
+@validate_params(
+    {
+        "remainder": [
+            StrOptions({"drop", "passthrough"}),
+            HasMethods(["fit", "transform"]),
+            HasMethods(["fit_transform", "transform"]),
+        ],
+        "sparse_threshold": [Interval(Real, 0, 1, closed="both"), None],
+        "n_jobs": [Integral, None],
+        "verbose": ["verbose"],
+        "verbose_feature_names_out": ["boolean"],
+    }
+)
```

Comment:
Can you revert this change because we already have a PR that is validating this function: https://github.com/scikit-learn/scikit-learn/pull/25897

Reply:
Yes, it's done